general_bounds.c

```diff
@@ -0,0 +1,45 @@
+#include "include/common.h"
```

Comment:
Conventionally the local includes come after the "system" includes. [Not a big deal, but we might as well do it!]

Reply:
All the examples were written following the opposite convention. I've now modified them as you've suggested. I suspect that they were written in that way because ``clang-format`` automatically resets the includes if you don't put a newline to divide them (alphabetical order?).

Reply:
Oh, if clang-format does that, we should leave them, otherwise we'll just end up fighting it.